src/reporter.ts

```diff
@@ -91,6 +91,7 @@ export default class SauceReporter extends SummaryFormatter {
 
   logTestCase(testCaseFinished: { testCaseStartedId: string }) {
     const testCaseAttempt = this.eventDataCollector.getTestCaseAttempt(testCaseFinished.testCaseStartedId);
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
Why disable? Preferably, eslint continues to warn us.

Reply:
Just add a "warn" rule like so https://github.com/saucelabs/sauce-testcafe-runner/blob/71f68ccb12898d64153241689a4d562292f6c7d5/.eslintrc#L12-L12

Reply:
Added.